$NetBSD: patch-bg,v 1.1 2006/01/18 23:24:12 rillig Exp $

The two structures are not meant to be empty, but its contents is
unspecified.

--- src/iquals.y.orig	2003-09-10 20:52:05.000000000 +0200
+++ src/iquals.y	2006-01-16 14:47:48.191242700 +0100
@@ -102,8 +102,8 @@ void fatal(const char *format, ...)
 }
 
 /* HACK:  We should really put a better interface in quals.c */
-struct Store {};
-struct Aloc {};
+struct Store;
+struct Aloc;
 void propagate_store_cell_forward(store s, aloc al) {}
 void propagate_store_cell_backward(store s, aloc al) {}
 void report_qerror(location loc, severity sev, const char *format, ...) {}
